src/Authoring/WinRT.SourceGenerator/WinRT.SourceGenerator.csproj

```diff
@@ -15,6 +15,7 @@
     <Description>C#/WinRT Authoring Source Generator Preview $(VersionString)</Description>
     <AssemblyTitle>C#/WinRT Authoring Source Generator Preview v$(VersionString)</AssemblyTitle>
     <Copyright>Copyright (c) Microsoft Corporation. All rights reserved.</Copyright>
+	  <IsRoslynComponent>true</IsRoslynComponent>
```

Comment:
What is the implication of this and nit: identing.

Reply:
I think that's just to enable the improved debugging experience, [see here](https://github.com/dotnet/roslyn-sdk/issues/850).